plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -541,10 +541,12 @@ function ListMenuItem:update()
 end
 
 function ListMenuItem:paintTo(bb, x, y)
-    -- We may get non-integer x or y (see mosaicmenu.lua)
-    -- Make them integer:
-    x = math.floor(x)
-    y = math.floor(y)
+    -- We used to get non-integer x or y that would cause some mess with image
+    -- inside FrameContainer were image would be drawn on top of the top border...
+    -- Fixed by having TextWidget:updateSize() math.ceil()'ing its length and height
+    -- But let us know if that happen again
+    assert(x == math.floor(x))
```

Comment:
OK, rebased with logger.err().
I'd rather not pop stuff up on the screen, as the PaintTo may be called a lot of times in a row.